Ines/ines.js

```diff
@@ -0,0 +1,25 @@
+// Q1. In one night, a movie theater sells tickets for 6450 dollars. Each ticket costs 15 dollars. How many tickets were sold?
+let movieTickets = 6450;
+let ticketCosts = 15;
+console.log(movieTickets / ticketCosts + " movie theater tickets were sold");
+
+// Q2. Sylvia's income is 500 dollars per week. How much does Sylvia makes every year?
+let weeks = 4 * 12;
```

Comment:
The second answer is wrong. There is 52 weeks per year and not 48!
The solution is :
let y = 500 * 52 which the result is 26k. But your result is 24k a year!